.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+**/coala-log
```

Comment:
1. Maybe call the file `coala.log`?
2. If you want to ignore a file regardless where it lies, you don't have to prepend `**`